src/pages/myPosts.tsx

```diff
@@ -25,56 +24,56 @@ export default function Home() {
 
   const [posts, setPosts] = useState<Post[] | null>(null);
 
-  function handleDeletePost(toDeletePost: Post): void {
-    setPosts(
-      (posts) =>
-        posts && posts.filter((post) => post.postId !== toDeletePost.postId)
-    );
-  }
-
   useEffect(() => {
     async function getPostData() {
       if (!userStatus.user) return;
       GetCurrentUserPosts(userStatus.user, supabaseClient)
         .then((p) => {
           setPosts(p);
-        }).catch((err) => {
-          console.log(err)
-          return
         })
+        .catch((err) => {
+          console.log(err);
+          return;
+        });
     }
 
     getPostData();
   }, [userStatus.user, supabaseClient]);
 
   if (userStatus.isLoading) return <Loading />;
   if (!userStatus.user) {
-    router.push(PagePaths.login);
+    router.push(PAGE_PATHS.LOGIN);
+    return;
+  }
+  if (!userStatus.user.isVerified) {
+    router.push(PAGE_PATHS.HOME)
     return;
   }
   if (posts == null) return <Loading />;
   return (
     <>
       <Navbar />
-      <Stack
-        spacing="40px"
-        sx={{
-          display: "flex",
-          alignItems: "center",
-          width: "100%",
-          flexDirection: "column",
-          padding: "30px",
-        }}
-      >
-        <Typography variant="h1">My post</Typography>
 
-        {posts.map((item, index) => (
-          <Box width="60vw" key={index}>
-            <MyPostCard post={item} handleDeletePost={handleDeletePost} />
-          </Box>
-        ))}
+      <Stack spacing={4} alignItems="center">
+        {/* Page header */}
+        <Box sx={{ marginTop: "3vh" }}>
+          <Typography variant="h1">My post</Typography>
+        </Box>
+        <Grid
+          container
+          justifyContent="space-between"
+          rowSpacing={6}
+          style={{ width: "80vw", marginTop: -6 }}
+        >
+          {posts.map((item, index) => (
+            <Grid item key={index} xs={5.75}>
+              <MyPostCard post={item} />
+            </Grid>
+          ))}
+        </Grid>
       </Stack>
-      {userStatus.user.isVerified &&
+
+      {userStatus.user.isVerified && (
         <Link href={"createPost"} color="inherit">
           <Fab
```

Comment:
Can you make it square like this?
<img width="85" alt="image" src="https://user-images.githubusercontent.com/62459518/226836688-5e5e942b-8a69-4f54-aade-d08e361506ca.png">

Reply:
Redesigned